guides/models/finding-records.md

```diff
@@ -32,25 +32,25 @@ this.store.findAll('blog-post') // => GET /blog-posts
   });
 ```
 
-Use [`store.peekAll()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/peekAll?anchor=peekAll) to retrieve all of the records for a given type that are already loaded into the store, without making a network request:
+Utiliser [`store.peekAll()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/peekAll?anchor=peekAll) pour récupérer tous les _records_ d'un type donné qui sont déjà chargé dans le _store_, sans déclencher une requête réseau:
 
 ```javascript
 let blogPosts = this.store.peekAll('blog-post'); // => no network request
 ```
 
-`store.findAll()` returns a `PromiseArray` that fulfills to a `RecordArray` and `store.peekAll` directly returns a `RecordArray`.
+`store.findAll()` retourne une `PromiseArray` qui est résolue avec un `RecordArray` et `store.peekAll` retourne directement un `RecordArray`.
 
-It's important to note that `RecordArray` is not a JavaScript array, it's an object that implements [`MutableArray`](https://api.emberjs.com/ember/release/classes/MutableArray).
-This is important because, for example, if you want to retrieve records by index,
-the `[]` notation will not work--you'll have to use `objectAt(index)` instead.
+Il est important de noter que `RecordArray` n'est pas un tableau JavaScript, c'est un objet qui implémente [`MutableArray`](https://api.emberjs.com/ember/release/classes/MutableArray).
+C'est important car, par exemple, si vous voulez récupérer des _records_ via leur index,
+la notation `[]` ne fonctionnera pas --vous devez utiliser `objectAt(index)` à la place.
 
-### Querying for Multiple Records
+### Requêter plusieurs _records_
 
-EmberData provides the ability to query for records that meet certain criteria.
-Calling [`store.query()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/query?anchor=query) will make a `GET` request with the passed object serialized as query params.
-This method returns a `PromiseArray` in the same way as `findAll`.
+EmberData offre la possibilité de rechercher des _records_ qui répondent à certains critères.
+Appler [`store.query()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/query?anchor=query) fera une requête `GET` avec l'objet sérialisé en tant que paramètre de requête.
```

Comment:
Here, I understand "passed object" as "the object passed to `store.query()`" so I think the term passed has to be translated to make the sentence clear.

I would also precise the (_query params_) because this term is going to be very common when working with EmberData.

```suggestion
Appelez [`store.query()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/query?anchor=query) fera une requête `GET` avec l'objet qui lui est passé sérialisé en tant que paramètre de requête (_query params_).
```